assets/styles/style.css

```diff
@@ -1,36 +1,69 @@
 .topic h1 {
-	font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-	font-size: 30px;
-	margin-bottom: 20px;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 30px;
+  margin-bottom: 20px;
 }
 
 #background {
-	background-image: image("coder.jpg");
-	background-size: cover;
-	background-position: center;
+  background: linear-gradient(#7777, #7890), url("../images/coder.png");
+  background-size: cover;
+  background-position: center;
+  color: white;
 }
```

Comment:
Your background image does not contain the page review it, please. Also the image position is fixed and the form scroll upon it.